.gitignore

```diff
@@ -13,6 +13,7 @@ _build
 
 # Generated merlin file.
 .merlin
+./**/.merlin
```

Comment:
Pattern `.merlin` should already ignore all files (at any depth).

Reply:
done